### PR TITLE
Translate scene barrier builder

### PR DESCRIPTION
**Pull request by Oshibuki**
#2

### CODE REVIEW

File: docs/content/schinese/Authoring Mission Scenes/Script Components/scene_barrier_builder.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+- 随心所欲的创建路径
 ![](/img/barrier_builder/create_path.png)
 
 
-- Press "Enable Barrier Build" chekbox on path inspector, it will create your barrier entity for you.
+- 在路径查看器中勾选"Enable Barrier Build"选择框, 它将为您创建屏障实体。
 ![](/img/barrier_builder/create_barrier.png)
 
-- You can go to that entity with "Go to Entity" Button and chnage its options like height, the entity named as "path_barrier_PATHNAME"
+- 您可以使用"Go to Entity"按钮转到该实体，并更改其选项，例如高度，该实体名为"path_barrier_PATHNAME"
```
Comment (Bisstocuz) [inline docs/content/schinese/Authoring Mission Scenes/Script Components/scene_barrier_builder.md:28]: 这个 PATHNAME 应该是（解释用途）变量名，可译

File: docs/content/schinese/Authoring Mission Scenes/Script Components/scene_barrier_builder.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
...
-- Create a path, you can use the button on toolbar
+- 添加路径，可使用工具条上按钮
 ![](/img/barrier_builder/add_path.png)
 
-- Give a name to path
+- 命名路径
 ![](/img/barrier_builder/name_path.png)
 
-- Build your path as you wish
+- 随心所欲的创建路径
```
Comment (Bisstocuz) [inline docs/content/schinese/Authoring Mission Scenes/Script Components/scene_barrier_builder.md:21]: 语法错误，“的”替换成“地”

**Review by Bisstocuz — CHANGES_REQUESTED** [review]

**Comment by Oshibuki** [comment]
@Bisstocuz resolved

**Review by Bisstocuz — APPROVED** [review]
Able to be merged.

**Pull request merged**